src/cml.js

```diff
@@ -502,7 +510,7 @@ class CML {
     const driver = this.getDriver();
     const {
       remote = GIT_REMOTE,
-      globs = ['dvc.lock', '.gitignore'],
+      globs = [],
```

Comment:
this makes sense to me, but does it break any old workflows?

Reply:
1. It doesn't break any old workflows because it never worked: previously, calling `cml pr create` without arguments just displayed the help text, so these defaults were being always ignored.
2. It conflicts with the new functionality introduced on this pull request.
3. This default should never have specified there, but in the `yargs` builder.